src/cdk/activation.ts

```diff
@@ -25,26 +37,39 @@ export async function activate(activateArguments: { extensionContext: vscode.Ext
         treeDataProvider: explorer,
         showCollapseAll: true
     })
+    activateArguments.extensionContext.subscriptions.push(view)
 
     // Indicates workspace includes a CDK app and user has expanded the Node
-    view.onDidExpandElement(e => {
-        if (e.element.contextValue === 'awsCdkAppNode') {
-            ext.telemetry.record(getTelemetryEvent('appExpanded'))
+    const appNodeExpanded = view.onDidExpandElement(e => {
+        if (e.element.contextValue === APP_NODE_CONTEXT_KEY) {
+            ext.telemetry.record(getTelemetryEvent(TelemetryEventTypes.APP_EXPANDED))
         }
     })
+    activateArguments.extensionContext.subscriptions.push(appNodeExpanded)
 
     // Indicates CDK explorer was disabled
-    vscode.workspace.onDidChangeConfiguration(e => {
-        const explorerEnabled = 'aws.cdk.explorer.enabled'
-        if (e.affectsConfiguration(explorerEnabled) && !vscode.workspace.getConfiguration().get(explorerEnabled)) {
-            ext.telemetry.record(getTelemetryEvent('explorerDisabled'))
+    const explorerDisabled = vscode.workspace.onDidChangeConfiguration(e => {
+        if (
+            e.affectsConfiguration(EXPLORER_ENABLED_CONFIG_KEY) &&
+            !vscode.workspace.getConfiguration().get(EXPLORER_ENABLED_CONFIG_KEY)
+        ) {
+            ext.telemetry.record(getTelemetryEvent(TelemetryEventTypes.EXPLORER_DISABLED))
         }
     })
+    activateArguments.extensionContext.subscriptions.push(explorerDisabled)
 
-    activateArguments.extensionContext.subscriptions.push(view)
+    const explorerReEnabled = vscode.workspace.onDidChangeConfiguration(e => {
```

Comment:
I think both onDidChangeConfiguration could be handled in the same callback. Registering more handlers might be less performant.